features/step_definitions/timehelpers_steps.rb

```diff
@@ -1,24 +1,39 @@
+# frozen_string_literal: true
+
+After do
+  travel_back
+end
+
+# Wrapper for travel_to but implementing safe nested traveling
+def safe_travel_to(time, &block)
+  previous_time = Time.zone.now
+  travel_back
+  travel_to(time, &block)
+ensure
+  travel_to(previous_time) if block_given?
+end
+
 def access_user_sessions
-  UserSession.where("revoked_at is null").each {|e| e.update_attribute(:accessed_at, Time.zone.now)}
+  UserSession.where("revoked_at is null").each {|session| session.update({ accessed_at: Time.zone.now} ) }
```

Comment:
Spaces seem a bit off.

```suggestion
  UserSession.where('revoked_at is null').each { |session| session.update({ accessed_at: Time.zone.now }) }
```